corehq/apps/smsbillables/models.py

```diff
@@ -286,20 +287,26 @@ def create(cls, message_log, api_response=None):
 
         country_code, national_number = get_country_code_and_national_number(phone_number)
 
-        billable.gateway_fee = SmsGatewayFee.get_by_criteria(
-            backend_api_id,
-            direction,
-            backend_instance=backend_instance,
-            country_code=country_code,
-            national_number=national_number,
-        )
-        if billable.gateway_fee is not None:
-            conversion_rate = billable.gateway_fee.currency.rate_to_default
-            if conversion_rate != 0:
-                billable.gateway_fee_conversion_rate = conversion_rate
+        backend = SMSBackend.get(backend_instance)
```

Comment:
I think this is fine - it's a simple get, and I'd prefer to not clutter the SMSLog as it's already got a lot of properties.

But we should also check here to make sure that backend_instance is not None before doing this lookup (it's not typically populated for inbound sms).  We can assume that if backend_instance is None that it's a global backend, otherwise we do the lookup and check is_global.

(The build failure also looks real and might be related to this)